src/components/NurseryWithdrawals/NurseryWithdrawals.tsx

```diff
@@ -115,32 +116,76 @@ export default function NurseryWithdrawals(props: NurseryWithdrawalsProps): JSX.
         values: filters.fromNurseryIds.map((id) => id.toString()),
       };
     }
+    if (nurseryFilter) {
```

Comment:
for all these filters, can we simply push as part of the if (filters.xyz && filters.xyz.length > 0) check?

```
if (filters.fromNurseryIds && filters.fromNurseryIds.length > 0) {
  filterValueChildren.push({
  });
}
```

Reply:
Totally! 💯 Much better